language/move-lang/src/parser/ast.rs

```diff
@@ -148,6 +148,7 @@ pub struct FunctionSignature {
 pub enum FunctionVisibility {
     Public(Loc),
     Internal,
+    Script(Loc),
```

Comment:
MEGA NIT, swap the position of Internal and Script. I like to order enums like this with some semantic meaning (most to least visible here)